queries/gnuplot/highlights.scm

```diff
@@ -0,0 +1,301 @@
+; highlights.scm
+
+(comment) @comment @spell
+
+(identifier) @variable
+
+(unary_expression
+  _? @operator
+  (_)
+  _? @operator)
+
+(binary_expression
+  (_)
+  _ @operator
+  (_))
+
+(ternary_expression
+  condition: (_)
+  "?" @conditional.ternary
+  true: (_)
+  ":" @conditional.ternary
+  false: (_))
```

Comment:
Frankly for these I'd just list out the operators as string literals (look in any other "mainstream" language's highlights to see what I mean) since it'll capture them in contexts where parsing is incomplete as well, or have an operator field for every case (look at bash)